superset-frontend/src/dashboard/components/nativeFilters/FilterCard/index.tsx

```diff
@@ -43,15 +44,20 @@ export const FilterCard = ({
   return (
     <Popover
       placement={placement}
-      overlayClassName="filter-card-popover"
+      overlayStyle={{
+        width: '240px',
+        padding: 0,
+        borderRadius: `${theme.gridUnit}px`,
```

Comment:
`theme.borderRadius`

Reply:
Do we need the border radius at all here? can we rely on vanilla Ant Design?